exercises/poker/src/main/java/Poker.java

```diff
@@ -0,0 +1,12 @@
+import java.util.List;
```

Comment:
According to the [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#starter-implementations), since this exercise is after the first 20 exercises it shouldn't have any starter implementation. Instead there should just be a `.keep` file here :)